### DISCUSSION TITLE
ServiceNow connector - Table API vs IRE API

### DISCUSSION THREAD

**Discussion by wi5nia** [microsoft/PowerPlatformConnectors#3568] [Q&A]
Hi,

the ServiceNow connector uses the table API
```
"path": "/api/now/v2/table/@{encodeURIComponent('cmdb_ci_vm_instance')}"
```

Is it possible to modify this connector or maybe some other way to make it use the [IRE API](https://docs.servicenow.com/bundle/washingtondc-api-reference/page/integrate/inbound-rest/concept/c_IdentifyReconcileAPI.html), because it has a lot of business logic, rules which are not taken into consideration when  using the table API directly?

**Comment by troystaylor** [comment]
I would suggest building your own custom connector from scratch. The open-source connector for ServiceNow is quite old and it will be more reliable to use the ServiceNow API version you need.
